### ISSUE TITLE
I have stop using Charlatano about a month ago but if the cheat go detected in the future will I get ban?

### ISSUE THREAD

**Issue by tamal777** [Jire/Charlatano#442]

**Comment by tamal777** [comment]
anybody knows about it?

**Comment by OVN1X** [comment]
Probably not. I was using r3cheats for a while, and then stopped. A month later it was detected, but I got **no** ban. Neither did any of my mates using it at the same time as me.

**Comment by tamal777** [comment]
thanks :)

**Comment by lzienius** [comment]
It all depends on whether the cheat has been detected before you stopped using it. In most cases, a shadow-ban goes back two weeks to a month on rare occasions, before an official ban wave. However, in _almost every case_, not using a cheat for over a month usually means you will **NOT** get banned.

tl;dr - no, you're 99.999% good

**Comment by tamal777** [comment]
Thanks friend btw I believe in Jire his cheat is the best and will not be detect that easily.
